examples/static_react_task/webapp/src/components/core_components.jsx

```diff
@@ -16,12 +16,21 @@ function OnboardingComponent({ onSubmit }) {
         qualification for your task. Click the button to move on to the main
         task.
       </Directions>
-      <button
-        className="button is-link"
-        onClick={() => onSubmit({ success: true })}
+      <div
+        style={{
+          width: "100%",
+          padding: "1.5rem 0",
+          display: "flex",
+          justifyContent: "center",
+        }}
       >
-        Move to main task.
-      </button>
+        <button
+          className="button is-link"
+          onClick={() => onSubmit({ success: true })}
+        >
+          Move to Main Task
+        </button>
+      </div>
```

Comment:
I was peeved by the fact that the onboarding button was not centered on the page.

Reply:
Yeeeeaaah this could've been adjusted long ago hahaha